modules/cudf/src/column.cpp

```diff
@@ -292,7 +292,7 @@ Column::Column(CallbackArgs const& args) : Napi::ObjectWrap<Column>(args) {
 
   this->size_ = props.Get("length");
 
-  if (this->size_ <= 0) {
+  if (!props.Has("length")) {
     auto type = this->type();
     if (cudf::is_fixed_width(type)) {
```

Comment:
It's a little odd to `Get("length")` before checking `Has("Length")` I think this structure below matches the intent, but reads a little clearer to me 
```cpp
if (props.Has("length")) {
  this->size_ = props.Get("length");
} else { 
  auto type = this->type();
  ...
}

Reply:
fixed it